packages/hash/frontend/src/components/layout/MainContentWrapper.tsx

```diff
@@ -69,7 +74,169 @@ export const MainContentWrapper: FunctionComponent = ({ children }) => {
           </IconButton>
         </Tooltip>
       </Fade>
-      <Main sidebarOpen={sidebarOpen}>{children}</Main>
+      <Main sidebarOpen={sidebarOpen}>
+        {children}
+        <Box mb={6} />
+        <Box sx={{ display: "flex", alignItems: "flex-start" }}>
+          <Button variant="primary" size="large">
+            Primary Large
+          </Button>
+          <Box sx={{ mr: 2 }} />
+          <Button size="medium">Primary Medium Button</Button>
+          <Box sx={{ mr: 2 }} />
+          <Button size="small">Primary Small Button</Button>
+          <Box sx={{ mr: 2 }} />
+          <Button size="xs">Primary XS Button</Button>
+          <Box sx={{ mr: 2 }} />
+        </Box>
+        <br />
```

Comment:
Added this for testing purposes since we don't have storybook setup at the moment. Will remove once the PR is approved

Reply:
Hah, great minds… 

<img width="1072" alt="image" src="https://user-images.githubusercontent.com/719814/158235284-b2d2fb2f-1c3d-4a6f-9c41-252895ba419c.png">

(I've done similar while working on buttons on HASH.dev, still very much a work in progress)